pkg/gomod/parse_testcase.go

```diff
@@ -0,0 +1,388 @@
+package gomod
+
+import "github.com/aquasecurity/go-dep-parser/pkg/types"
+
+var (
```

Comment:
not sure about repo to pick for test cases, let me know if other repos are preferred.